checkov/circleci_pipelines/checks/ReverseShellNetcat.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+from typing import Any
+
+from checkov.circleci_pipelines.base_circleci_pipelines_check import BaseCircleCIPipelinesCheck
+from checkov.common.models.enums import CheckResult
+from checkov.yaml_doc.enums import BlockType
+import re
+
+
+class ReverseShellNetcat(BaseCircleCIPipelinesCheck):
+    def __init__(self) -> None:
+        name = "Suspicious use of netcat with IP address"
+        id = "CKV_CIRCLECIPIPELINES_5"
+        super().__init__(
+            name=name,
+            id=id,
+            block_type=BlockType.ARRAY,
+            supported_entities=['jobs.*.steps[]']
+        )
+
+    def scan_conf(self, conf: dict[str, Any]) -> tuple[CheckResult, dict[str, Any]]:
+        if "run" not in conf:
+            return CheckResult.PASSED, conf
+        run = conf.get("run", "")
+        if type(run) == dict:
+            run = run.get("command", "")
+        if re.search(r'(nc|netcat) (\d{1,3}).(\d{1,3}).(\d{1,3}).(\d{1,3})', run):
```

Comment:
```suggestion
        if isinstance(run, dict):
            command = run.get("command", "")
        if re.search(r'(nc|netcat) (\d{1,3}).(\d{1,3}).(\d{1,3}).(\d{1,3})', command):
```
`isinstance` is most of the time the preferred way of checking a variable type, because it also handles subclasses.
Better not to reuse a variable name, if the content is actually changed